hydromt/data_source/__init__.py

```diff
@@ -1,21 +1,19 @@
 """DataSources responsible for validating the DataCatalog."""
 
 # importing all data sources for discovery, factory needs to be imported last.
-from .data_source import DataSource, SourceMetadata  # noqa: I001
+from .data_source import DataSource  # noqa: I001
```

Comment:
don't think the noqa is necessary anymore since we export it in the `__all__`  right?

Reply:
noqa is needed still, as the order of imports is important